packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -78,6 +78,70 @@
             />
           </td>
         </tr>
+        <tr v-if="runningOnEos">
+          <th scope="col" class="oc-pr-s" v-text="eosPathLabel" />
+          <td>
+            <div class="oc-flex oc-flex-middle oc-flex-between oc-width-1-1">
+              <div class="oc-flex oc-flex-middle">
+                <p
+                  ref="filePath"
+                  v-oc-tooltip="file.path"
+                  class="oc-my-rm oc-text-truncate"
+                  v-text="file.path"
+                />
+              </div>
+              <oc-button
+                oc-tooltip="Copy EOS path"
+                appearance="raw"
+                aria-label="Copy EOS path"
+                class="oc-files-private-link-copy-url"
+                :variation="copiedEos ? 'success' : 'passive'"
+                @click="copyEosPathToClipboard"
+              >
+                <span text="EOS Path" />
```

Comment:
Please use a `label` prop and `this.$gettext()` here

Reply:
Same as https://github.com/owncloud/web/pull/6959#discussion_r874611497.